### ISSUE TITLE
Do we trust Keybase?

### ISSUE THREAD

**Issue by drzax** [abcnews/editorslab-2017#4]
This is part of [some feedback](https://gist.github.com/WPettersson/1be2b55a59a7c60980cb666b5e9621ad) from @WPettersson.

> You're trusting keybase.io. Be aware of this, it's part of your trust model. Look at how they do security. Will you check on them from time to time? Will end users/whistleblowers be able to trust them easily? What security precautions do they use?

From what I know of keybase, I don't think we're using the service in a way which would introduce further risk.  I wonder if @keybase has any thoughts?

Do we trust them? I think so.

**Comment by WPettersson** [comment]
Again I don't expect you to have problems here, but it might be worthwhile having some process to check this every week/month/6 months. Keybase is something I trust right now, too, and I think you are using keybase correctly. But what if they up and sell everything in 6 months to some other organisation? Again, this could be as simple as spending 5 minutes once a month just double checking that no one is writing stories about Keybase selling themselves, or selling data, or being hacked.

**Comment by drzax** [comment]
Yep, that sounds reasonable. We've also considered that we don't actually *need* keybase. For anyone who is comfortable managing their own keys @colingourlay has suggested we make an advanced signup option possible where a public key can be supplied. 

What keybase *does* provide though, is some level of identity verification. So I think we'd want to think about messaging around non-keybase linked accounts.
